scripts/install/functions.sh

```diff
@@ -382,3 +413,30 @@ function cmd_help {
     echo " stop"
     echo " help"
 }
+
+function create_npmrc {   
+    local NPMRC_CONTENT="always-auth=${NPM_ALWAYS_AUTH}\n@spartacus:registry=${NPM_URL}\n$(echo ${NPM_URL} | sed 's/https://g'):_auth=${NPM_TOKEN}\nemail=${NPM_EMAIL}\n"
+    if [ -z "$NPM_TOKEN" ] ; then
+        echo "NPM_TOKEN is empty"
+    fi
+    if [ -z "$NPM_EMAIL" ] ; then
+        echo "NPM_EMAIL is empty"
+    fi    
+    echo "creating .npmrc file in ${1} folder"    
+    printf $NPMRC_CONTENT > .npmrc
+    echo "Spartacus registry url for ${1} app: $(npm config get '@spartacus:registry')"   
+}
+
+function remove_npmrc {   
+    if [ -f "./.npmrc" ]; then
+        echo 'removing .npmrc file'
+        rm -rf .npmrc
+    fi
+}
+
+function remove_npm_token {
```

Comment:
maybe some people might the token in the config.default.sh? should we check and remove it from there too just in case?

Reply:
good idea!